substrate/primitives/application-crypto/src/bandersnatch.rs

```diff
@@ -41,14 +46,26 @@ impl RuntimePublic for Public {
 		sp_io::crypto::bandersnatch_generate(key_type, seed)
 	}
 
-	/// Dummy implementation. Returns `None`.
-	fn sign<M: AsRef<[u8]>>(&self, _key_type: KeyTypeId, _msg: &M) -> Option<Self::Signature> {
-		None
+	fn sign<M: AsRef<[u8]>>(&self, key_type: KeyTypeId, msg: &M) -> Option<Self::Signature> {
```

Comment:
I'll comment on the other pull request to ask @davxy if it is ok to add these given that we are not adding a new host function.